tools/run_tests/run_interop_tests.py

```diff
@@ -1029,5 +1096,8 @@ def aggregate_http2_results(stdout):
   dockerjob.finish_jobs([j for j in server_jobs.itervalues()])
 
   for image in docker_images.itervalues():
-    print('Removing docker image %s' % image)
-    dockerjob.remove_image(image)
+    if not args.manual_run:
+      print('Removing docker image %s' % image)
+      dockerjob.remove_image(image)
+    else:
+      print('Preserving docker image: %s' % image)
```

Comment:
Can we save the list of images produced in this cycle to a docker_images file so it can be 1) cleaned up manually; and/or 2) uploaded to GCR by another script?